src/modules/partition/core/PartitionCoreModule.cpp

```diff
@@ -881,12 +882,18 @@ PartitionCoreModule::initLayout( const QVariantList& config )
             maxSizeString = CalamaresUtils::getString( pentry, "maxSize" );
         }
 
+        if ( pentry.contains( "features" ) )
+        {
+            featureMap = pentry.values( "features" ).at(0).toMap();
```

Comment:
You might want to use just `value()` there, or also `CalamaresUtils::getSubMap()` -- the latter is supposed to be a convenience function, although I don't know how convenient it is.

Reply:
I used `getSubMap()`. I change the code a bit to remove the `if` statement.